using-timescaledb/compression.md

```diff
@@ -0,0 +1,417 @@
+9.6 in oeder to use compression you must be using postgres 10 or 11 or higher.
+
+
+As of version 1.5,TimescaleDB supports the ability to natively compress data. This 
+function does not require the use of any specific file system or external software, 
+and as you will see in the coming paragraphs, is simple to set up and customizable 
+by the user. 
+
+Prior to reading this guide, we recommend taking a look at our architecture section 
+to learn more about how compression works. At a high level, TimescaleDB’s built-in 
+job scheduler framework will asynchronously convert recent data from an uncompressed 
+row-based form to a compressed columnar form across chunks of TimescaleDB hypertables. 
+Once a chunk is old enough, the chunk will be transactionally converted from the 
+row to columnar form. 
+
+This section will walk through the concepts and help you understand some of the 
+benefits and limitations of native compression. We will also walk you through the 
+basics of setting this up for use in your environment. 
+
+>:TIP: As with any type of data altering operation, we would suggest backing up 
+your important data prior to implementing compression.
+
+You can compress data as it comes into TimescaleDB in one of two ways:
+
+Policy-based compression: Set up automatic compression of data once it has reached 
+a certain age.
+
+Manually compress chunks: Use explicit commands that will compress chunks that you 
+specify.
+
+Before we start, we will give you a high-level overview of how compression works 
+using an example that has been implemented based on policy. 
+
+With regards to compression, a chunk can be in one of three states: active (uncompressed), 
+compression candidate (uncompressed), compressed. Active chunks are those that are 
+currently ingesting data. Due to the nature of the compression mechanism, they cannot 
+effectively ingest data while compressed.
+
+![alt text](https://assets.timescale.com/images/diagrams/compression_diagram.png "compress")
+
+In this case, the data in Position 0 (which is our active chunk) will always be 
+uncompressed, and the current active chunk will not be a candidate for compression 
+in order to protect the system’s ability to perform high volume/high velocity ingestion. 
+Once data moves out of Position 0 (the active chunk) and into something greater 
+than or equal to Position 1, which is now considered historical data, it then becomes 
+a compression candidate (it can now be compressed manually or via policy).  
+
+In this example we have chosen NOT to compress the chunk in position 1, however 
+all full chunks (position 1-4) are candidates for compression. As you can see in 
+this example we have chosen to wait until a chunk reaches Position 2 (in this case 
+the chunk is 3 days old) before we decide to apply compression,meaning chunks 3 days 
+and older will be compressed.
+
+
+### Preparing Hypertable for Compression [](prepare)
```

Comment:
More descriptive anchor names (this is really important to set properly now, since links that will go out into the wild will use these anchors, so we don't want to change unless absolutely necessary)

Reply:
Throughout the doc